config/consensus.go

```diff
@@ -231,6 +231,9 @@ type ConsensusParams struct {
 	// program in bytes
 	MaxAppProgramLen int
 
+	// maximum total length of an application's programs (approval + clear state)
```

Comment:
I think it needs clarification about extra pages, like "total combined length per page" or something